pkgs/love2d/example/assets/love_example_browser/examples/012_cursor_change.lua

```
@@ -0,0 +1,11 @@
+-- Example: Use an Image as cursor
+
+function love.load()
+	-- Load the "cursor" (with default hot spot 0, 0)
+	cursor = love.mouse.newCursor("assets/love-cursor.png")
```

Luacheck (1.2.0)

[warning] setting non-standard global variable 'cursor'

(W111)